test/Microsoft.SqlTools.ServiceLayer.UnitTests/QueryExecution/Execution/ServiceIntegrationTests.cs

```diff
@@ -131,12 +133,12 @@ public void GetSqlTextFromInvalidType()
             // ... Mock up an implementation of ExecuteRequestParamsBase
             // ... Create a query execution service without a connection service or workspace
             //     service (we won't execute code that uses either
-            var mockParams = new Mock<ExecuteRequestParamsBase>().Object;
+            var invalidParams = new InvalidParams() { OwnerUri = "" };
```

Comment:
This was failing because the mocked object had a null OwnerUri, but given that we assume OwnerUri is a valid string in many other places in the code already it seemed reasonable to just set it to empty here instead of updating the function to handle a null Uri.